rest/rest-resources/src/main/resources/OSGI-INF/blueprint/service.xml

```diff
@@ -74,18 +74,20 @@ limitations under the License.
     <bean id="adjunctResourceBean" class="org.apache.brooklyn.rest.resources.AdjunctResource"/>
     <bean id="apidocResourceBean" class="org.apache.brooklyn.rest.resources.ApidocResource"/>
     <bean id="applicationResourceBean" class="org.apache.brooklyn.rest.resources.ApplicationResource"/>
+    <bean id="bundlesResourceBean" class="org.apache.brooklyn.rest.resources.BundleResource"/>
```

Comment:
Very minor: I'd go for `bundleResourceBean` to be consistent with all the others. Then good to merge!

Reply:
@aledsage updated